about.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Holi 2023</title>
+  <link rel="stylesheet" href="./css/about.css">
+  <link rel="stylesheet" href="./css/index.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="nav-bar">
+        <a href="index.html"><img src="./images/Logo.png" alt="holi-logo" class="logo"></a>
+        <button class="hambug-menu">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+        <ul class="mobile-menu">
+          <li><a class="menuItem" href="index.html">Home</a></li>
+          <li><a class="menuItem" href="about.html">About</a></li>
+          <li><a class="menuItem" href="#">Programs</a></li>
+          <li><a class="menuItem" href="#">Gallery</a></li>
+          <li><a class="menuItem" href="#">Come Celebrate</a></li>
+        </ul>
+
+        <ul class="nav-links">
+          <li class="links"><a class="alinks" href="about.html">About</a></li>
+          <li class="links"><a class="alinks" href="#">Programs</a></li>
+          <li class="links"><a class="alinks" href="#">Gallery</a></li>
+          <li class="links"><a class="alinks borderLink" href="#">Come Celebrate</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <section class="aboutHeadContent">
```

Comment:
- [ ] I must commend your choice of color 💯 . I will recommend you follow the background as specified in the design guideline for both mobile and desktop views. Kindly ensure that your design (background, font, color etc) matches the design provided for you because when provided with a design in a real working environment, you will be expected to implement the designs to be pixel-perfect. Please check it out 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/217949763-e3926ec0-966d-41a7-8227-2c039e12c3d5.png)

Design guidelines 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/217949713-9e13cab1-4e34-4417-bf3b-873de8c4079f.png)

Reply:
@vickymarz  Please😥, my topic is about Holi festival in India if I change the background image then it's not suitable for my design topic. It will look different and the message will not be passed.